### PR TITLE
Better gaslight

### PR DESCRIPTION
**Pull request by Soapy7261**
This uses a function for gaslight so we can have it know what emojis are on the server and have things generally look nicer

### CODE REVIEW

**Comment by Soapy7261** [comment]
And yes toothy, the code works on my other bot and tbh there's not much room for error

File: cogs/socials.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ async def gpt(self, ctx: discord.ApplicationContext, text: str):
         await ctx.defer()
         messages.reverse()
         url = "https://free.churchless.tech/v1/chat/completions"
-        gpthistory = [{"role": "system", "content": f"{data.gaslight} The user's name is {ctx.author.display_name}. Do not use the user's full name, use their call name derived from their full name."}]
+        gpthistory = [{"role": "system", "content": f"{get_gaslight(ctx.guild.emojis, ctx.author.display_name)}"}]
```
Comment (ToothyDev) [inline cogs/socials.py:264]: I doubt the code works on your machine because you never await the async get_gaslight function.

**Review by ToothyDev — CHANGES_REQUESTED** [review]
Fix the commented issue

**Comment by JoshuaSlui** [comment]
I do not review PRs regarding the GPT function. Please stick to requesting review from @ToothyDev from now on.
Thanks in advance.

**Comment by Soapy7261** [comment]
It auto requested you

**Review by ToothyDev — APPROVED** [review]

**Pull request merged**